README.md

```diff
@@ -118,14 +118,32 @@ accountsUIBootstrap3.setCustomSignupOptions = function() {
 }
 ```
 
+### Disable User Account Creation
+
+If you would like to prevent users from creating new accounts, you can choose to not show the account creation buttons as shown below.
+
+```javascript
+Accounts.ui.config({
+    forbidClientAccountCreation : true
+});
+```
+
+While this flag prevents the user from seeing the "Create Account" link, the server will still accept new accounts. You can prevent new account creation altogether by always returning false in the [account validation stage](http://docs.meteor.com/#/full/accounts_validatenewuser) on the server. Here is an example (place in server-side code):
```

Comment:
Actually, this is not true. If you run the `Accounts.ui.config` method both on the client and the server, it won't accept new accounts either, even if you try to do it through the browser console.